apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -42,7 +43,9 @@ export class FilesStorageUC {
 		private readonly previewService: PreviewService,
 		// maybe better to pass the request context from controller and avoid em at this place
 		private readonly em: EntityManager,
-		private readonly domainErrorHandler: DomainErrorHandler
+		private readonly domainErrorHandler: DomainErrorHandler,
+		private readonly instanceConfigService: InstanceConfigService,
+		private readonly schoolService: SchoolService
```

Comment:
The fileStorage mode should function independently of other services. It should resolve via “AuthorizationReferenceService”.